tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1630,19 +1718,25 @@ def adjust_brightness(image, delta):
   images, `delta` should be in the range `[0,1)`, as it is added to the image in
   floating point representation, where pixel values are in the `[0,1)` range.
 
+  Usage Example:
+    >>> x = [[[1.0, 2.0, 3.0],
+    ...       [4.0, 5.0, 6.0]],      
+    ...     [[7.0, 8.0, 9.0],      
+    ...       [10.0, 11.0, 12.0]]]
+    >>> tf.image.adjust_brightness(x, delta=0.1)
+    <tf.Tensor: shape=(2, 2, 3), dtype=float32, numpy=
+    array([[[ 1.1,  2.1,  3.1],
+            [ 4.1,  5.1,  6.1]],
+
```

Comment:
I think this empty line (and all others like it) should be removed, based on the error you see now:

```
..._impl.py", line 1726, in tensorflow.python.ops.image_ops_impl.adjust_brightness
Failed example:
    tf.image.adjust_brightness(x, delta=0.1)
Expected:
    <tf.Tensor: shape=(2, 2, 3), dtype=float32, numpy=
    array([[[ 1.1,  2.1,  3.1],
            [ 4.1,  5.1,  6.1]],
Got:
    <tf.Tensor: shape=(2, 2, 3), dtype=float32, numpy=
    array([[[ 1.1,  2.1,  3.1],
            [ 4.1,  5.1,  6.1]],

           [[ 7.1,  8.1,  9.1],
            [10.1, 11.1, 12.1]]], dtype=float32)>
```

Reply:
Yeah probably because that's the only thing that's different in expected/got results at the moment.

Reply:
I really hope that it works this time. Did the change -> https://github.com/tensorflow/tensorflow/pull/35388/commits/3a77a7cd8b3d8388f988ee3cedcfc0600bbe397f